sdk/cosmosdb/cosmos/README.md

```diff
@@ -213,6 +213,17 @@ The Azure SDKs are designed to support ES5 JavaScript syntax and [LTS versions o
 
 While working with Cosmos DB, you might encounter transient failures caused by [rate limits][cosmos_request_units] enforced by the service, or other transient problems like network outages. For information about handling these types of failures, see [Retry pattern][azure_pattern_retry] in the Cloud Design Patterns guide, and the related [Circuit Breaker pattern][azure_pattern_circuit_breaker].
 
+### Logging
+
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`. While using `AZURE_LOG_LEVEL` make sure to set it before logging library is initialized. Ideally pass it through command line, if using libraries like `dotenv` make sure such libraries are initialized before logging library.
```

Comment:
_Ideally pass it through command line, if using libraries like `dotenv` make sure such libraries are initialized before logging library._

separate these two sentences.  Ideally pass it through the command line. If you are using libraries such as `dotenv`  make sure to initialize before initializing the logging library

Reply:
Ack.